Sources/App/Extensions/Application+Static.swift

```diff
@@ -14,7 +14,7 @@ extension Application {
     
     // paths are always absolute, with a trailing slash
     struct Paths {
-        static let base: String = Environment.get("BASE_PATH")!
+        static let base: String = Environment.get("BASE_PATH")!.expandingTildeInPath.absolutePath
```

Comment:
We don't need the .absolutePath here, see my other comments, also please merge with the latest master, since we have to use the `Environment.fetch` method from now on. Thank you for your help.